tests/unit/components/DocumentationTopic.spec.js

```diff
@@ -368,6 +369,37 @@ describe('DocumentationTopic', () => {
     expect(hierarchy.exists()).toBe(false);
   });
 
+  it('only creates a "Mentioned In" section for non-module pages', () => {
+    const mentionSection = {
+      kind: PrimaryContent.constants.SectionKind.mentions,
+      mentions: [
+        'topic://foo',
+        'topic://bar',
+      ],
+    };
+
+    wrapper.setProps({
+      references: hierarchyItemsReferences,
+      role: TopicTypes.symbol,
+      symbolKind: SymbolKind.protocol,
+      primaryContentSections: [
+        mentionSection,
+        foo,
+      ],
+    });
+
+    expect(wrapper.find(PrimaryContent).props()).toHaveProperty('sections', [
```

Comment:
(Not a change you need to make, just pointing out that you can request specific props from the `wrapper.props` function, which can come in handy sometimes.)

```suggestion
    expect(wrapper.find(PrimaryContent).props('sections')).toEqual([
```